cms/modules/events/events_common.php

```diff
@@ -289,6 +289,78 @@ function showEventMap(){
 exit();
 }
 
+function register(){
+	$gcm_regid = $_POST["regId"];
+	$gcm_regid = mysql_real_escape_string($gcm_regid);
+	$result = mysql_query("INSERT INTO gcm_users(gcm_regid) VALUES('$gcm_regid')");
+		if ($result) {
```

Comment:
`mysql_query` always returns `true` if a *successful* insert is made, so the next `SELECT` query is excess.